cli/data.go

```diff
@@ -18,8 +18,11 @@ import (
 )
 
 const (
-	dataDir     = "data"
-	metadataDir = "metadata"
+	dataDir       = "data"
+	metadataDir   = "metadata"
+	maxRetryCount = 5
+	logEveryN     = 100
+	timeFormat    = "2006-01-02T150405.0000Z"
```

Comment:
I originally did this to get nanoseconds (but we have file IDs to enforce uniqueness) and not use colons (since the mac browser shows them as "/"), but better to use the same format we're using when capturing data. Updated to RFC3339.